atomate/vasp/fireworks/lobster.py

```diff
@@ -0,0 +1,118 @@
+# coding: utf-8
+"""
+This module defines fireworks  that can be used to handle Lobster calculations that are based on VASP wavefunctions.
+"""
+
+import logging
+import os
+from typing import List, Union
+
+from atomate.common.firetasks.glue_tasks import DeleteFiles, PassCalcLocs, DeleteFilesPrevFolder
+from atomate.vasp.config import DB_FILE
+from atomate.vasp.firetasks.glue_tasks import CopyVaspOutputs
+from atomate.vasp.firetasks.lobster_tasks import WriteLobsterinputfromIO, RunLobster, LobsterRunToDb
+from custodian.custodian import ErrorHandler, Validator
+from fireworks import Firework
+from pymatgen.core.structure import Structure
+
+__author__ = "Janine George, Guido Petretto"
+__email__ = 'janine.george@uclouvain.be, guido.petretto@uclouvain.be'
+
+LOBSTER_CMD = ">>lobster_cmd<<"
+VASP_OUTPUT_FILES_without_PPKI = ["OUTCAR", "vasprun.xml", "CHG", "CHGCAR", "CONTCAR", "DOSCAR", "EIGENVAL", "IBZKPT",
+                                  "OSZICAR", "PCDAT", "PROCAR", "REPORT", "WAVECAR", "XDATCAR"]
+
+MODULE_DIR = os.path.dirname(os.path.abspath(__file__))
+logger = logging.getLogger(__name__)
+
+
+class LobsterFW(Firework):
+    """
+    A Firework performs a Lobster calculation based on a previous static VASP calculation with specific configurations
+    """
+
+    def __init__(self, structure: Structure = None, name: str = "lobster_calculation", lobster_cmd: str = LOBSTER_CMD,
+                 db_file: str = DB_FILE, delete_wavecar: bool = False,
+                 delete_wavecar_previous_fw: bool = False,
+                 handler_group: Union[List[ErrorHandler], str] = "default",
+                 validator_group: Union[List[Validator], str] = "default",
+                 calculationtype: str = 'standard',
+                 parents: Union[List[Firework], Firework] = None,
+                 prev_calc_dir: str = None, prev_calc_loc: bool = True, user_supplied_basis: dict = None,
+                 lobsterin_key_dict: dict = None, lobstertodb_kwargs: dict = None,
+                 additional_outputs: List[str] = None, **kwargs):
+        """
+
+        Args:
+            structure (Structure): Structure object, will only be used to name firework
+            name (str): name of the firework
+            lobster_cmd (str): command to run lobster
+            db_file (str): address to db_file
+            delete_wavecar (bool): If True, WAVECAR will be deleted

```

Comment:
Could you be a little bit more specific about the difference between this argument and the next argument.

Reply:
will do.